### PR TITLE
Fix proto2 enum default values

### LINKED ISSUE #53: Bug: Enum default values in proto2
Proto2 doesn't require enums to be 0-indexed, and therefore protojure generates incorrect behaviour for e.g. 1-indexed enums.

### PR DESCRIPTION
**Pull request by sharrap**
Fixes #53

Changes the behaviour of enum defaults to always refer to the first element in the enum, rather than the element with index 0.

[This is still correct for proto3](https://developers.google.com/protocol-buffers/docs/proto3#default):

> For enums, the default value is the first defined enum value, which must be 0.

[While handling proto2 enums without a zero or with weird orderings](https://developers.google.com/protocol-buffers/docs/proto#optional):

> For enums, the default value is the first value listed in the enum's type definition.

### CODE REVIEW

**Comment by ghaskins** [comment]
LGTM.  I'll wait for your other MR to be merged first, then merge this one, and then cut a new release.  Thank you for the contribution!

File: test/protojure/plugin/core_test.clj [hunk 1]
```diff
@@ -40,6 +40,9 @@
     (require '[com.example.addressbook :as addressbook] :reload
              '[com.example.kitchensink :as example] :reload
              '[com.example.kitchensink.Greeter.server :as example-server] :reload
+             '[com.example.enum_proto :as enum-proto]
```
Comment (ghaskins) [inline test/protojure/plugin/core_test.clj:43]: I missed this the first time, but the lack of :reload may be a problem.  TBH, I've somewhat forgotten why we needed those but I do remember it was important.

Comment (sharrap) [reply test/protojure/plugin/core_test.clj]: Sounds good, added.

**Comment by ghaskins** [comment]
@sharrap thank you.  Both of your MRs are included in https://github.com/protojure/protoc-plugin/releases/tag/v0.9.4

**Pull request merged**